site/docs/concepts/modules.md

```diff
@@ -59,7 +59,7 @@ Name | Description | FybrikModule | Prerequisite
 [arrow-flight-module](https://github.com/fybrik/arrow-flight-module) | reading and writing datasets while performing data transformations | https://raw.githubusercontent.com/fybrik/arrow-flight-module/master/module.yaml |
 [airbyte-module](https://github.com/fybrik/airbyte-module) | reading datasets from data sources supported by the [Airbyte](https://airbyte.com) tool | https://raw.githubusercontent.com/fybrik/airbyte-module/main/module.yaml |
 [delete-module](https://github.com/fybrik/delete-module) | deletes s3 objects | https://raw.githubusercontent.com/fybrik/delete-module/main/module.yaml |
-[implicit-copy](https://github.com/fybrik/mover) | copies data between any two supported data stores, for example S3 and Kafka, and applies transformations. | https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-batch-module.yaml<br> <br>https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-stream-module.yaml | [`FybrikStorageAccount`](../../reference/crds#fybrikstorageaccount) resource deployed in the control plane namespace to hold the details of the storage which is used by the module for coping the data.
+[implicit-copy](https://github.com/fybrik/mover) | copies data between any two supported data stores, for example S3 and Kafka, and applies transformations. | https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-batch-module.yaml<br> <br>https://raw.githubusercontent.com/fybrik/data-movement-operator/master/modules/implicit-copy-stream-module.yaml | [`FybrikStorageAccount`](../../reference/crds#appfybrikiov1beta2) resource deployed in the control plane namespace to hold the details of the storage which is used by the module for coping the data.
```

Comment:
the word "FybrikStorageAccount" is cut in the table. Can you check if possible to wider the column?

Reply:
<nobr></nobr> seems to do the work.